Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -462,29 +496,30 @@ def select_item(self, item, state):
             self._re_enumerate_selections()
             self.selectionChanged.emit()
 
+    @staticmethod
+    def _create_path(item, path):
```

Comment:
This code is just factored out from lines 477 and 592. This change is not necessary; I've needed it in my initial attempt, but then kept it because it's better this way.